bundles/org.openhab.binding.yamahamusiccast/src/main/java/org/openhab/binding/yamahamusiccast/internal/model/DeviceInfo.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.yamahamusiccast.internal.model;
+
+import org.eclipse.jdt.annotation.*;
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * This class represents the DeviceInfo request requested from the Yamaha model/device via the API.
+ *
+ * @author Lennert Coopman - Initial contribution
+ */
+@NonNullByDefault
+public class DeviceInfo {
```

Comment:
DTOs shouldn't be annotated with NonNullByDefault, because it gives a false sense of safety, as the JSON serializer could set fields to null.

You could move this into a package called `dto` or append `DTO` to the class name to get rid of the checkstyle warning about missing NonNullByDefault annotation.

Reply:
Thanks for the tip, didn't knew that one. Moved it to package `dto`
I made a pull request at the same time to update documentation.